cms/static/js/views/move_xblock_list.js

```diff
@@ -127,7 +130,7 @@ function($, Backbone, _, gettext, HtmlUtils, StringUtils, XBlockUtils, MoveXBloc
                 this.parentInfo.parent.get('category'),
                 this.visitedAncestors[this.visitedAncestors.length - 2]
             );
-            this.childrenInfo.category = this.categoryRelationMap[this.parentInfo.category];
+            this.childrenInfo.category = this.categoryRelationMap[this.parentInfo.category] || 'component';
```

Comment:
Are we missing any other parent category ? I'm asking because of explicit `'component'`.